src/plugins/ima/index.js

```diff
@@ -1035,6 +1020,25 @@
       }
     }.bind(this);
 
+    var setAdMuted = function(mute) {
+      if (mute) {
+        addClass_(this.muteDiv, 'ima-muted');
+        removeClass_(this.muteDiv, 'ima-non-muted');
+        this.adsManager.setVolume(0);
+        // Bubble down to content player
+        this.player.muted(true);
+        this.adMuted = true;
+        this.sliderLevelDiv.style.width = "0%";
+      } else {
+        addClass_(this.muteDiv, 'ima-non-muted');
+        removeClass_(this.muteDiv, 'ima-muted');
+        this.adsManager.setVolume(this.player.volume());
+        // Bubble down to content player
+        this.player.muted(false);
+        this.adMuted = false;
+        this.sliderLevelDiv.style.width = this.player.volume() * 100 + "%";
```

Comment:
I assume this line needs to be changed as well?

Reply:
way? this your code I just refactored it

Reply:
@yaniv-li It doesn't mean that it's 100% correct :)

Reply:
True, my bad.